components/org.wso2.micro.integrator.security/src/main/java/org/wso2/micro/integrator/security/user/core/hybrid/JdbcHybridRoleManager.java

```diff
@@ -290,7 +291,13 @@ public String[] getHybridRoles(String filter) throws UserStoreException {
     public String[] getUserListOfHybridRole(String roleName) throws UserStoreException {
 
         if (UserCoreUtil.isEveryoneRole(roleName, realmConfig)) {
-            return userRealm.getUserStoreManager().listUsers("*", -1);
+            if (Boolean.parseBoolean(System.getProperty("NonUserCoreMode"))) {
```

Comment:
shall we use the constant defined for this property

Reply:
Required changes were addressed by commit [e2c6818]([e2c6818](https://github.com/wso2/micro-integrator/pull/2771/commits/e2c681846d8fbe08ea30fac8d6ccf9e0f777a623))